packages/core/admin/admin/src/pages/Marketplace/components/NpmPackageCard.tsx

```diff
@@ -272,28 +275,17 @@ const CardButton = ({
       return (
         <Tooltip
           data-testid={`tooltip-${pluginName}`}
-          label={
-            !versionRange
-              ? formatMessage(
-                  {
-                    id: 'admin.pages.MarketPlacePage.plugin.version.null',
```

Comment:
We should delete this string from the translations as well, unless we plan to use it again in the near future and reference it in a ticket so we don't forget it

Reply:
no I don't think we need it, I think the strategy is to be more strict anyways, instead of saying we don't know